Form/Type/BooleanType.php

```diff
@@ -81,7 +81,9 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'choice';
+        return class_exists('Symfony\Component\Form\Extension\Core\Type\ChoiceType') ?
```

Comment:
Here is [the correct method](https://github.com/sonata-project/SonataAdminBundle/commit/9f152086379ce9cb7b7a0b642b8c0a15cd1817b0). Can you please try that and report back?